src/librustdoc/passes/collect_intra_doc_links.rs

```diff
@@ -986,6 +987,11 @@ impl LinkCollector<'_, '_> {
                 }
             }
 
+            if path_str.contains(&['<', '>'].as_slice()) {
```

Comment:
Remove `&` -> potentially unnecessary

```suggestion
            if path_str.contains(['<', '>'].as_slice()) {
```